docs/_toc.yaml

```diff
@@ -27,3 +27,7 @@ toc:
   path: /cirq/research/qaoa/hardware_grid_circuits
 - title: "Routing with tket"
   path: /cirq/research/qaoa/routing_with_tket
+
+- heading: "Quantum Chess"
+- title: "Concepts"
+  path: /cirq/research/quantum_chess/concepts
```

Comment:
Are there more docs coming to this section?

Reply:
Yes, Chris Cantwell is providing a tutorial on the Rest API and client.  I am currently reviewing them outside of github, so hopefully a PR coming soon.